### PR TITLE
A Helm Chart for Openldap

### PR DESCRIPTION
**Pull request by keyolk**
A Helm Chart for OpenLDAP
for testing LDAP Auth.

including OpenLDAP and phpLDAPadmin.

### CODE REVIEW

File: openldap/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+PhpLdapAdmin:
+  Image: "docker.io/osixia/phpldapadmin"
+  ImageTag: "0.6.12"
+  ImagePullPolicy: "Always"
+  Component: "phpadmin"
+
+  Replicas: 1
+
+  NodePort: 31080
+  #LdapEndpoint: kube-1.local.io:30389
```
Comment (coffeepac) [inline openldap/README.md]: can we get a comment explaining why this is commented out?  I suspect its as an example but a comment describing why would be best as I may be wrong.

Comment (keyolk) [inline openldap/README.md]: well the endpoint is consumed by phpLDAPadmin
if there is not exact configuration
it will use ldap's service name to access.

Comment (keyolk) [inline openldap/README.md]: it could be useless. how do you think ?

Comment (coffeepac) [inline openldap/README.md]: ahh, okay.  I didn't see the default operator.

good to go!

Comment (keyolk) [inline openldap/README.md]: Thanks feedback.

**Pull request merged**